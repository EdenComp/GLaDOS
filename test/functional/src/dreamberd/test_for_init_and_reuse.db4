int i = 0;

for (i = 10; i < 15; i += 2) {
    print(i + "\n");
}
i += 10;
print("Final value: " + i + "\n");
