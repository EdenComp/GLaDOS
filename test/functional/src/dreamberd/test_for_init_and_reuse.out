10
12
14
Final value: 26
